editor/src/components/assets.ts

```diff
@@ -42,11 +42,18 @@ export function getAllProjectAssetFiles(
   return allProjectAssets
 }
 
-function getSHA1Checksum(contents: string): string {
+function getSHA1Checksum(contents: string | Buffer): string {
   return new sha1().update(contents).digest('hex')
 }
 
-export function getProjectContentsChecksums(tree: ProjectContentTreeRoot): GithubChecksums {
+export function inferGitBlobChecksum(buf: Buffer): string {
+  return getSHA1Checksum(Buffer.concat([Buffer.from(`blob ${buf.byteLength}\0`), buf]))
```

Comment:
My eyes! Please can you add a comment (+ link if relevant) explaining this (I remember you mentioning it, but that will absolutely be forgotten in the near future)

Reply:
https://github.com/concrete-utopia/utopia/pull/2933/commits/01521400b0a41bb5a0ab56d1ae07135a03603701 + https://github.com/concrete-utopia/utopia/pull/2933/commits/b282ef3ca384327bbb3e139968052f00a6606e41